Formula/pocl.rb

```diff
@@ -0,0 +1,39 @@
+class Pocl < Formula
+  desc "Portable Computing Language"
+  homepage "http://portablecl.org"
+  url "http://portablecl.org/downloads/pocl-3.0.tar.gz"
+  sha256 "a3fd3889ef7854b90b8e4c7899c5de48b7494bf770e39fba5ad268a5cbcc719d"
+  license "MIT"
+  head "https://github.com/pocl/pocl.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "opencl-headers" => :build
+  depends_on "pkg-config" => :build
+  depends_on "hwloc"
+  depends_on "llvm"
+  depends_on "ocl-icd"
+
+  fails_with gcc: "5" # LLVM is built with GCC
+
+  def install
+    args = ["-DPOCL_INSTALL_ICD_VENDORDIR=#{prefix}/etc/OpenCL/vendors"]
+    # Add RPATH for libraries installed into #{lib}/pocl directory
+    args << "-DCMAKE_INSTALL_RPATH=@loader_path;@loader_path/.." if OS.mac?
```

Comment:
Still checking on this. Could alternatively use absolute paths and let brew handle rewrite, but I think we prefer paths that are already relocatable.
```suggestion
    # Add RPATH for libraries installed into #{lib}/pocl directory
    args << "-DCMAKE_INSTALL_RPATH=#{lib};#{lib}/pocl" if OS.mac?
```

Reply:
Looks like needs help on Linux too, so might just use absolute path rather than switching between `@loader_path` and `$ORIGIN` logic.
```
Full linkage --test pocl output
  Missing libraries:
    unexpected (libpocl-devices-basic.so)
```